include/sdlmain.h

```diff
@@ -148,20 +143,10 @@ struct SDL_Block {
 		double dpi_scale = 1.0;
 		bool fullscreen = false;
 
-		// This flag indicates, that we are in the process of switching
-		// between fullscreen or window (as oppososed to changing
-		// rendering size due to rotating screen, emulation state, or
-		// user resizing the window).
-		bool switching_fullscreen = false;
-		// Lazy window size init triggers updating window size and
-		// position when leaving fullscreen for the first time.
-		// See FinalizeWindowState function for details.
-		bool lazy_init_window_size = false;
```

Comment:
Very nice to have these eliminated!

Reply:
Yeah, but if we got rid of it then I would say we'll need to do serious regression testing on KDE and remember to use debug builds during testing (I remember we had some failing assert on vertical screens some time ago).